setup_mysql.sh

```diff
@@ -46,8 +30,6 @@ run_mysql_secure_installation() {
   # Use HERE document vs. as a straight multiline string - to avoid quoting issues
   echo
   echo "GETTING assigned root password from file..."
```

Comment:
Do you still need this `echo` after removing the assignment to `MYSQL_ROOT_PWD` that used to be on the next line?

Reply:
@kcharter 
I have several lines as checks/debugging throughout the scripts, because I still find it hard to see that things are running correctly with the barrage of output in the terminal as the scripts run.

But, I remove the lines you pointed out, which also shorten the code a bit, and edited Lines 201 & 202, which come before the call of `run_mysql_secure_installation`. Lines 201+ are now:
```
  echo "Temporary root password for MySQL exists, so you have NOT run mysql_secure_installation yet."
  echo "WILL NOW RUN run mysql_secure_installation & set password to use the one from ${MYSQL_PW_FILE}."
  echo
  run_mysql_secure_installation
```